r/vignettes/install.Rmd

```diff
@@ -7,6 +7,12 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
+TL;DR
+
+In most cases, `install.packages("arrow")` should just work. There are things you can do to make the installation faster (see below). If, for any reason, it doesn't work, set the environment variable `ARROW_R_DEV=true` in your `.Renviron`, retry, and share the logs with us.
+
+----
```

Comment:
I haven't built this locally myself; what do these dashes here look like when we build it?